packages/utils/rollup.npm.config.mjs

```diff
@@ -1,4 +1,6 @@
+import replace from '@rollup/plugin-replace';
 import { makeBaseNPMConfig, makeNPMConfigVariants } from '@sentry-internal/rollup-utils';
+import packageJson from './package.json' with { type: 'json' };
```

Comment:
Does this work in our GH action (currently node 18)?

Reply:
I am pretty sure this was added in node 18.20.0 and we are on 18.20.3 😎